app/controllers/groups_controller.rb

```diff
@@ -3,7 +3,7 @@ class GroupsController < ApplicationController
 
   # GET /groups or /groups.json
   def index
-    @groups = Group.all
+    @groups = Group.all.where(user_id: current_user)
```

Comment:
Alternatively, because you have 

https://github.com/agustinbooth/Contact-your-contacts/blob/2b761dd322f9e2f4525affc01437282918710419/app/models/user.rb#L8

You could write

```suggestion
    @groups = current_user.groups
```